### ISSUE TITLE
Updating `device_info_plus` dependency constraints

### ISSUE THREAD

**Issue by utkarsh1097** [kyle-seongwoo-jun/flutter_apple_product_name#5]
## Issue faces

```
Because apple_product_name >=1.3.0 depends on device_info_plus ^8.0.0 and MyTestApp depends on device_info_plus 9.0.2, apple_product_name >=1.3.0 is forbidden.
So, because MyTestApp depends on apple_product_name 1.4.0, version solving failed.
```
## Specifics

[device_info_plus:9.0.0](https://pub.dev/packages/device_info_plus/versions/9.0.0) was first released over a month ago with changes that should not directly impact any of this package's functionalities. We will really appreciate if the dependency constraints can also be updated in this package's `pubspec.yaml`.

Thank you very much for this useful package!

**Comment by kyle-seongwoo-jun** [comment]
Thanks for reporting this! I'll update it ASAP :)

**Comment by kyle-seongwoo-jun** [comment]
@utkarsh1097, thank you for reporting this issue! 

I've just released version [2.0.0](https://pub.dev/packages/apple_product_name/versions/2.0.0) which should resolve the dependency issue. Please check it out and let me know if it works for you. 

Thank you 😊
